caching-service/src/main/java/org/zowe/apiml/caching/service/vsam/config/VsamConfiguration.java

```diff
@@ -27,9 +27,12 @@ public class VsamConfiguration {
     private final VsamConfig vsamConfig;
     private final VsamInitializer vsamInitializer;
 
+
     @ConditionalOnProperty(name = "caching.storage.mode", havingValue = "vsam")
     @Bean
     public Storage vsam(MessageService messageService, EvictionStrategyProducer evictionStrategyProducer) {
-        return new VsamStorage(vsamConfig, vsamInitializer, ApimlLogger.of(VsamStorage.class, messageService), evictionStrategyProducer);
+        var logger = ApimlLogger.of(VsamStorage.class, messageService);
```

Comment:
Could be injected I guess

Reply:
is it better?

Reply:
well I think it is if this needs to have a test, if it's declared a dependency it's easier to mock and test

Reply:
do we need a test for this bean?

Reply:
it's up to you, since it's printing a message maybe, but in this scenario it's probably not needed